Application.hs

```diff
@@ -93,22 +93,12 @@ makeFoundation appSettings = do
 
   where
     getOAuthKeys :: String -> IO OAuthKeys
-    getOAuthKeys plugin = do
-        envClientId     <- lookupEnv $ plugin <> "_OAUTH_CLIENT_ID"
-        envClientSecret <- lookupEnv $ plugin <> "_OAUTH_CLIENT_SECRET"
-
-        case (envClientId, envClientSecret) of
-            (Just clientId, Just clientSecret) ->
-                return OAuthKeys
-                    { oauthKeysClientId     = pack clientId
-                    , oauthKeysClientSecret = pack clientSecret
-                    }
-
-            _ -> error
-                    $  plugin
-                    <> "_OAUTH_CLIENT_ID or "
-                    <> plugin
-                    <> "_OAUTH_CLIENT_SECRET not set"
+    getOAuthKeys plugin = OAuthKeys
```

Comment:
Is `pluginName` better, or am I stuck in Ruby naming and I should get hip to Haskell's single-variable names again?